lambda/getBinInterval.ts

```diff
@@ -0,0 +1,32 @@
+import {
+	QueryCommand,
+	type TimestreamQueryClient,
+} from '@aws-sdk/client-timestream-query'
+import { parseResult } from '@bifravst/timestream-helpers'
+
+export const getBinInterval =
+	(ts: TimestreamQueryClient, dbName: string, tableName: string) =>
+	async ({
+		binIntervalMinutes,
+		durationHours,
+		iccid,
+	}: {
+		binIntervalMinutes: number
+		durationHours: number
+		iccid: string
+	}): Promise<Record<string, unknown>[]> => {
```

Comment:
**praise**: good example of function signature with dependency injection and runtime parameters.